sdk/dataprotection/Azure.ResourceManager.DataProtectionBackup/src/Customized/Models/AdhocBackupRules.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using Azure.Core;
+
+namespace Azure.ResourceManager.DataProtectionBackup.Models
+{
```

Comment:
This is a work around for this issue: https://github.com/Azure/autorest.csharp/issues/2808